pyfftw/pyfftw.pyx

```diff
@@ -1394,6 +1405,10 @@ cdef class FFTW:
         copy the returned array.
         '''
 
+        if ortho and normalise_idft:
+            raise ValueError('Invalid options: ortho and normalise_idft cannot'
+                             ' both be True.')
+
```

Comment:
hmmm I'm not super keen on this - it's a bit mucky to have a requirement for exclusive arguments. However, I can't see how to do it cleanly whilst maintaining backwards compatibility.